8-show-manifest-and-values/README.md

```diff
@@ -0,0 +1,333 @@
+
+Below is the intended help text for `zarf dev show-manifests`. `zarf dev show-values-files` will include the same flags.
+```
+Usage:
+  zarf dev show-manifests [ DIRECTORY ] [flags]
+
+Flags:
+      --create-set stringToString   Specify package variables to set on the command line (KEY=value) (default [])
+      --deploy-set stringToString   Specify deployment variables to set on the command line (KEY=value) (default [])
+  -f, --flavor string               The flavor of components to include in the resulting package (i.e. have a matching or empty "only.flavor" key)
+      --kube-version                Override the default helm template KubeVersion when performing a package chart template
+      --confirm                     Confirms command without prompting. Skips prompts to configure variables.
+```
+
+Below is the intended help text for `zarf package show-manifests`. `zarf package show-values-files` will include the same flags.
+```
+Usage:
+  zarf package show-manifests [ PACKAGE_SOURCE ] [flags]
+
+Flags:
+      --set stringToString          Specify deployment variables to set on the command line (KEY=value) (default [])
+      --kube-version                Override the default helm template KubeVersion when performing a package chart template
+      --components                  Comma-separated list of components whose manifests should be displayed.  Adding this flag will skip the prompts for selected components.  Globbing component names with '*' and deselecting 'default' components with a leading '-' are also supported
+      --confirm                     Confirms command without prompting. Skips prompts to configure variables and select optional components
+```
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this ZEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1
+
+As a creator of Zarf packages, I want to make sure the variables in my package can get templated properly for the expected values of the deployers. I want to check this for both manifests and values files so I run `zarf dev show-manifests path/to/package-dir --deploy-set=MY_VAR=my-val` and `zarf dev show-values-files path/to/package-dir --deploy-set=MY_VAR=my-val`
+
+#### Story 2
+
+As a deployer of Zarf packages, I want to check that the variables I intend to deploy my package with are getting properly templated for both manifests and values files before I deploy so I run `zarf package show-manifests zarf-package-podinfo-amd64.tar.zst -set=MY_VAR=my-val` and `zarf package show-values-files zarf-package-podinfo-amd64.tar.zst --set=MY_VAR=my-val`
```

Comment:
I don't see how this story is different from the first one, other than the first is using directory, and this a zst file.

Reply:
I wanted to highlight that this will be used both before a package is created, and after a package is created. I added the different flags that the different commands have to make the stories more clear.